script/eslint.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ts-node
+
+import ChildProcess = require('child_process')
+import Path = require('path')
+
+const ESLINT_ARGS = [
+  '--cache',
+  '--rulesdir=./eslint-rules',
+  './{script,eslint-rules}/**/*.{j,t}s?(x)',
+  './tslint-rules/**/*.ts',
+  './app/{src,typings,test}/**/*.{j,t}s?(x)',
+  ...process.argv.slice(2),
+]
+const opts = {
+  stdio: 'inherit',
+}
+
+if (process.env.CI) {
+  ChildProcess.spawn(
+    Path.resolve(__dirname, '..', 'node_modules', '.bin', 'eslint'),
```

Comment:
`eslint_d` runs the linter in the background, enabling it to load up much faster since ESLint doesn’t have to load all its modules every time you want to lint.